### DISCUSSION TITLE
How can i get the number of endrow?  我怎么才能知道最后一行的值是多少

### DISCUSSION THREAD

**Discussion by ynf2039** [sourcelair/xterm.js#5149] [Q&A]
When I select the last row and call xterm.getSelectionPosition() method, I can get a value of endrow, how can I get this value when the last row is not selected?
当我选择了最后一行并且调用getSelectionPosition方法时，我能得到一个endrow的值，我怎么做才能在不选择最后一行时获取到这个值？

**Comment by jerch** [comment]
Your question is unclear. `getSelectionPosition()` returns you the selected rectangle in col/row coordinates as a upper-left and lower-right point. If your selection ends before the last row, then the returned lower-right point is off by the amount of lines not selected, thus getting the same value is `lower_right_point + not_selected_rows_below`.

Also you are on an xtermjs version < v5.0.0, you should upgrade to a newer version as the API you mention is outdated.
